### PR TITLE
Added time types to response bodies

### LINKED ISSUE #5: Dealing with Time fields
Response bodies contain some fields related with time. For example, `StartDate` and `Duration`. Right now we handle these fields as strings but for the next major version it's better to parse this data to `time.Time` type.

### PR DESCRIPTION
**Pull request by gunjan01**
This PR aims to modify response bodies to accommodate time related fields. Fixes #5 

@rumyantseva  PTAL. Let me know if I have missed something :)

### CODE REVIEW

File: velobike/history.go [hunk 1]
```diff
@@ -8,36 +10,36 @@ type HistoryService struct {
 // History describes the body of ride/history method response.
 type History struct {
 	Items          []HistoryItem `json:"Items,omitempty"`
-	TotalRidesTime *string       `json:"TotalRidesTime,omitempty"`
+	TotalRidesTime time.Time     `json:"TotalRidesTime,omitempty"`
```
Comment (rumyantseva) [inline velobike/history.go]: From my point of view, there is some sense to use pointers (as it was implemented for strings) -- if the field is not presented it in the response, it's better to have it as a nil pointer, not as a zero value for `time.Time`. What do you think?

**Comment by gunjan01** [comment]
@rumyantseva I agree. Had missed that point . Have updated my pull.. Please take a look now

**Comment by gunjan01** [comment]
@rumyantseva Hi ! Any updates on this ?

**Comment by rumyantseva** [comment]
Hi! Sorry, I didn't have time to check if it actually works (having no tests is a big problem, yeah).
But let's merge it (why not).

**Comment by rumyantseva** [comment]
Thank you very much!

**Comment by gunjan01** [comment]
@rumyantseva I will try to write some tests and send a PR.Can you please create an issue for the same ? :)

**Pull request merged**